src/main/java/com/uwetrottmann/tmdb2/entities/Account.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 Nikolas Mavropoylos
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.uwetrottmann.tmdb2.entities;
+
+public class Account {
+
+    public static class Avatar {
+        public static class GRAvatar {
```

Comment:
Globally Recognized Avatars
As concerns of the coding standards, over the years i learned, you have to capitalize, every first letter of a word. In no case i'm not a Java developer, what you see is a port of my .NET knowledge in Java. I Started java, 1 year ago, as a necessity. I Ignored the fact that all the service classes are Tmdb, which for me it should be TMDb , TheMovieDatabase.

I'm not sure if the coding standards in java are so off, than .NET's but if they does, please let me know. I'm new to java and i'm willing to learn and adapt.